esp-hal/src/spi/mod.rs

```diff
@@ -1,5 +1,10 @@
-//! Serial Peripheral Interface
+//! Serial Peripheral Interface (SPI)
 //!
+//! ## Overview
+//! The Serial Peripheral Interface (SPI) is a synchronous serial interface
+//! useful for communication with external peripherals.
```

Comment:
```suggestion
//! used for communication with external peripherals.
```

Reply:
This was taken from the TRM 
![image](https://github.com/esp-rs/esp-hal/assets/12926049/8d5e682e-b935-454f-9c03-28ebe2d976f4)